scripts/auto-rebase/last_lvms_rebase.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash -x
-./scripts/auto-rebase/rebase-lvms.sh to "registry.redhat.io/lvms4/lvms-operator-bundle:v4.15.0"
+./scripts/auto-rebase/rebase-lvms.sh to "quay.io/lvms_dev/lvms4-lvms-operator-bundle:v4.17.0-18"
```

Comment:
Can we update https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/rebase_job_entrypoint.sh#L49 to this image or it will fail due to access?

Reply:
changed to dynamically resolve based on the version (candidate or not) and also made it auto update itself on rebase so that this may never be forgotten again by me :)